ValidationLibrary.Rules/HasDescriptionRule.cs

```diff
@@ -43,6 +44,14 @@ public Task<ValidationResult> IsValid(IGitHubClient client, Repository gitHubRep
             return Task.FromResult(new ValidationResult(RuleName, "Add description for this repository.", isValid, DoNothing));
         }
 
+        public Dictionary<string, string> GetConfiguration()
+        {
+            return new Dictionary<string, string>
+            {
+                 { "PullRequestTitle", RuleName }
```

Comment:
This rule doesn't create a Pull Request, so `PullRequestTitle` doesn't make sense here